### ISSUE TITLE
Delate upload directory

### ISSUE THREAD

**Issue by joel688** [SalahHW/webserv#30]
Now im creating an upload directory where the uploaded content will be store. Maybe we have to destroy the directory with make clean. What do you think?

**Comment by SalahHW** [comment]
I couldn’t find any exact references to this topic in the Nginx documentation. Perhaps we should set up a test server with Nginx to observe its behavior and replicate it as closely as possible.

Let me know what you think about this approach.

**Comment by joel688** [comment]
Bro wtf no need to read nginx doc just as we can upload with post method we have to store it somewhere and i propose to delete the directory where we store uploaded things with make clean

**Comment by SalahHW** [comment]
Yes, I understood that. It is obvious, I am just suggesting that we should observe Nginx's behavior so we can maybe discover more about how it handles similar situations. This could help us ensure our implementation aligns as closely as possible with its behavior, which is one of our project goals.

Additionally, observing Nginx might reveal subtle details or edge cases that we might otherwise overlook. Let me know if you agree with this approach or if you have another suggestion.

**Comment by joel688** [comment]
Nginx have no Makefile so no way to replicate.

**Comment by SalahHW** [comment]
That is exactly what I was thinking about. So, I imagine Nginx has preconfigured folders for temporary storage, and that's all? Could you clarify where exactly the uploaded files are stored, or if there’s a specific behavior we should be aware of?
